cpu/atmega_common/periph/gpio.c

```diff
@@ -133,7 +135,10 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
     uint8_t pin_num = _pin_num(pin);
 
     if ((_port_num(pin) == PORT_D && pin_num > 3)
-         || (_port_num(pin) == PORT_E && pin_num < 4)
+//#if !defined(__AVR_ATmega328P__)
+#if defined (PORTE)
```

Comment:
it looks weird but PORTX notation is how are ports defined in avr/libc as
#define PORTB _SFR_IO8(0x05)
and PORT_X is the common way to enumarate them RIOT in perip_cpu.h.
I don't how to do better ..